caddy/app.go

```diff
@@ -27,8 +27,11 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var replayCache outline.ReplayCache
+
 func init() {
 	caddy.RegisterModule(OutlineApp{})
+	replayCache = outline.NewReplayCache(0)
```

Comment:
See my comment to use the registration type in the other PR. You can inject the replay cache directly into the `New` function

Reply:
Done.